src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/ResourceGroup.kt

```diff
@@ -0,0 +1,7 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
+
+data class ResourceGroup(
+        var resources: List<Resource>,
+        var title: String
+        // TODO: Icon?
+)
```

Comment:
Should [BookElement](https://github.com/WycliffeAssociates/otter-common/pull/93/files#diff-e8e9f76e7dd4be5765c7e426af406bdc) have an Observable of these instead of Resources?

Reply:
Even after our discussion yesterday, we might want to revisit this question. I believe BookElement would only have one ResourceGroup associated with it (since there would only be front matter associated with the Chapter), and it sounds like we want to wait for all the Resources in the ResourceGroup to be fetched before we display them anyway, so this might make sense

Reply:
I don't think so, no

Reply:
response was to Russell, sorry, I didn't have the up to date page when I commented